src/Model/TransportModel/tsp1apt1.f90

```diff
@@ -2463,53 +2356,96 @@ subroutine apt_stor_term(this, ientry, n1, n2, rrate, &
     real(DP), intent(inout), optional :: hcofval
     real(DP) :: v0, v1
     real(DP) :: c0, c1
+    !
     n1 = ientry
     n2 = ientry
     call this%get_volumes(n1, v1, v0, delt)
     c0 = this%xoldpak(n1)
     c1 = this%xnewpak(n1)
-    if (present(rrate)) rrate = -c1 * v1 / delt + c0 * v0 / delt
-    if (present(rhsval)) rhsval = -c0 * v0 / delt
-    if (present(hcofval)) hcofval = -v1 / delt
+    if (present(rrate)) then
+      rrate = (-c1 * v1 / delt + c0 * v0 / delt) * this%eqnsclfac
+    end if
+    if (present(rhsval)) rhsval = -c0 * v0 * this%eqnsclfac / delt
+    if (present(hcofval)) hcofval = -v1 * this%eqnsclfac / delt
     !
-    ! -- return
+    ! -- Return
     return
   end subroutine apt_stor_term
 
+  !> @brief Account for mass or energy transferred to the MVR package
+  !<
   subroutine apt_tmvr_term(this, ientry, n1, n2, rrate, &
                            rhsval, hcofval)
-    class(GwtAptType) :: this
+    ! -- modules
+    ! -- dummy
+    class(TspAptType) :: this
     integer(I4B), intent(in) :: ientry
     integer(I4B), intent(inout) :: n1
     integer(I4B), intent(inout) :: n2
     real(DP), intent(inout), optional :: rrate
     real(DP), intent(inout), optional :: rhsval
     real(DP), intent(inout), optional :: hcofval
+    ! -- local
     real(DP) :: qbnd
     real(DP) :: ctmp
+    !
+    ! -- Calculate MVR-related terms
     n1 = this%flowbudptr%budterm(this%idxbudtmvr)%id1(ientry)
     n2 = this%flowbudptr%budterm(this%idxbudtmvr)%id2(ientry)
     qbnd = this%flowbudptr%budterm(this%idxbudtmvr)%flow(ientry)
     ctmp = this%xnewpak(n1)
-    if (present(rrate)) rrate = ctmp * qbnd
+    if (present(rrate)) rrate = ctmp * qbnd * this%eqnsclfac
     if (present(rhsval)) rhsval = DZERO
-    if (present(hcofval)) hcofval = qbnd
+    if (present(hcofval)) hcofval = qbnd * this%eqnsclfac
     !
-    ! -- return
+    ! -- Return
     return
   end subroutine apt_tmvr_term
 
+  !> @brief Account for mass or energy transferred to this package from the
+  !! MVR package
+  !<
+  subroutine apt_fmvr_term(this, ientry, n1, n2, rrate, &
+                           rhsval, hcofval)
+    ! -- modules
+    ! -- dummy
+    class(TspAptType) :: this
+    integer(I4B), intent(in) :: ientry
+    integer(I4B), intent(inout) :: n1
+    integer(I4B), intent(inout) :: n2
+    real(DP), intent(inout), optional :: rrate
+    real(DP), intent(inout), optional :: rhsval
+    real(DP), intent(inout), optional :: hcofval
+    !
+    ! -- Calculate MVR-related terms
+    n1 = ientry
+    n2 = n1
+    if (present(rrate)) rrate = this%qmfrommvr(n1) ! presumably in terms of energy already for heat transport???
```

Comment:
Is this ready to go into develop?

Reply:
After discussing with @aprovost-usgs, yes, this is ready to go into develop.  We wanted to leave a note for ourselves to double check values when bringing in GWE later.